mcp-github-example/src/main/java/dev/langchain4j/example/mcp/github/McpGithubToolsExample.java

```diff
@@ -0,0 +1,66 @@
+package dev.langchain4j.example.mcp.github;
+
+import dev.langchain4j.mcp.McpToolProvider;
+import dev.langchain4j.mcp.client.DefaultMcpClient;
+import dev.langchain4j.mcp.client.McpClient;
+import dev.langchain4j.mcp.client.transport.McpTransport;
+import dev.langchain4j.mcp.client.transport.stdio.StdioMcpTransport;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.openai.OpenAiChatModel;
+import dev.langchain4j.service.AiServices;
+import dev.langchain4j.service.tool.ToolProvider;
+
+import java.util.List;
+
+public class McpGithubToolsExample {
+
+    /**
+     * This example uses the GitHub MCP server to showcase how
+     * to use an LLM to summarize the last commits of a public GitHub repo.
+     * Being a public repository (the LangChain4j repository is used as an example), you don't need any
+     * authentication to access the data.
+     * <p>
+     * Running this example requires Docker to be installed on your machine,
+     * because it spawns the GitHub MCP Server as a subprocess via Docker:
+     * `docker run -i mcp/github`.
+     * <p>
+     * You first need to build the Docker image of the GitHub MCP Server that is available at `mcp/github`.
+     * See https://github.com/modelcontextprotocol/servers/tree/main/src/github to build the image.
+     * <p>
+     * The communication with the GitHub MCP server is done directly via stdin/stdout.
+     */
+    public static void main(String[] args) throws Exception {
+
+        ChatLanguageModel model = OpenAiChatModel.builder()
+                .apiKey(System.getenv("OPENAI_API_KEY"))
+                .modelName("gpt-4o-mini")
+                .logRequests(true)
+                .logResponses(true)
+                .build();
+
+        McpTransport transport = new StdioMcpTransport.Builder()
+                .command(List.of("/usr/local/bin/docker", "run", "-i", "mcp/github"))
```

Comment:
```suggestion
                .command(List.of("/usr/local/bin/docker", "run", "-e", "GITHUB_PERSONAL_ACCESS_TOKEN", "-i", "mcp/github"))
```

Reply:
I would pass the environment variable in this way to Docker (tested on Docker Desktop for Mac)